src/common/utils/general/general.ts

```diff
@@ -40,6 +40,10 @@ export function isBlank(str: string): boolean {
   return str.length === 0;
 }
 
+export function isFiniteNumber(n: number): boolean {
+  return typeof n === "number" && isFinite(n) && !isNaN(n);
```

Comment:
is a typeof operator necessary here?

Reply:
plywood declares it's numeric values as numbers but it sometimes returns `null` (it's not compiled with `--strict`) so we need check to be sure

the case was with formulas and division - when denominator was zero, plywood returned null